buildlib/pr/main.yml

```diff
@@ -37,16 +37,16 @@ resources:
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/rhel7.2/builder:mofed-5.0-1.0.0.0
       options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_VOLUMES)
     - container: rhel82
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/rhel8.2/builder:mofed-5.0-1.0.0.0
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/x86_64/rhel8.2/builder-mofed-5.0-1.0.0.0:1
```

Comment:
Where are Dockerfiles for the new images?

Reply:
Those were never found. Installed CUDA over existing images instead of building from scratch.

Reply:
As we discussed investigate whether it's safe to use `docker commit` (potential docker layer issues). It may make sense to create a Dockerfile and use hpcx images as base.